iocore/hostdb/HostDB.cc

```diff
@@ -2771,11 +2586,14 @@ ParseHostLine(char *l)
   int n_elts = elts.Initialize(l, SHARE_TOKS);
   // Elements should be the address then a list of host names.
   // Don't use RecHttpLoadIp because the address *must* be literal.
-  HostFilePair item;
-  if (n_elts > 1 && 0 == item.ip.load(elts[0]) && !item.ip.isLoopback()) {
+  IpAddr ip;
+  if (n_elts > 1 && 0 == ip.load(elts[0]) && !ip.isLoopback()) {
     for (int i = 1; i < n_elts; ++i) {
-      item.name = elts[i];
-      HostFilePairs.push_back(item);
+      ts::ConstBuffer name(elts[i], strlen(elts[i]));
+      // If we don't have an entry already (host files only support single IPs for a given name)
```

Comment:
Is this true? I've done quite a bit of research on this and can't verify that accuracy of that claim. I wrote the previous version on the presumption you could have round robin data for a host by using multiple lines with distinct IP addresses and the same FQDN.